modules/tinymce/src/core/test/ts/browser/dom/DomUtilsTest.ts

```diff
@@ -788,3 +789,70 @@ UnitTest.asynctest('browser.tinymce.core.dom.DomUtilsTest', function (success, f
     success();
   }, failure);
 });
+
+UnitTest.test('DOMUtils.get - by id in head', () => {
```

Comment:
I'm not sure I like this approach for a number of reasons:

1. It's inconsistent with how all other tests are implemented in this file, which will no doubt end up causing confusion.
2. We normally wouldn't have a `finally`, as everywhere else we leave the element in the page for failed attempts normally so that you can inspect it to find out why it didn't work. So this should probably only clean up if it's successful, so we could remove the try/finally block.

Reply:
I can remove the finally. I don't see why I would need a pipeline or asyncTest for synchronous behaviours. Are you suggesting I use LegacyUnit?

Reply:
Yes, that's what every other/existing test in this file uses (even if async isn't needed), so we should keep it consistent.

Reply:
But don't we want to move away from LegacyUnit... I mean, ... it's legacy :)

Reply:
How about I remove LegacyUnit from the rest of the tests in this file?

Reply:
Yeah, that'd work but it's not a small bit of work. Another option would be to put the new tests in a new file if possible.

Reply:
Nothing that can't be smashed with IntelliJ refactoring tools and a few search-replaces. I've converted it all. Except the test labels - that would take forever.

Reply:
Oh wow, thanks for that Dylan! Looks a lot better now. 👍